src/snapred/backend/data/LocalDataService.py

```diff
@@ -842,7 +842,7 @@ def _writeDefaultDiffCalTable(self, runNumber: str, useLiteMode: bool):
         indexer = self.calibrationIndexer(runNumber, useLiteMode)
         version = indexer.defaultVersion()
         grocer = GroceryService()
-        filename = Path(grocer._createDiffcalTableWorkspaceName("default", useLiteMode, version) + ".h5")
+        filename = Path(grocer.createDiffcalTableWorkspaceName("default", useLiteMode, version) + ".h5")
```

Comment:
I don't get this: why does "default" subsume the <run number> position (especially as "default" as a string is _not_ a legitimate <run number>), but not just use the `VERSION_DEFAULT` which is provided for that reason particularly?  (And then use the _real_ <run number> which might be useful information.  What am I missing here?)

Reply:
I realized later that possibly there's no meaningful \<run number\> where this is actually required.  I would like to see these "default" names centralized somewhere (we have the same requirement for \< version \> ), but that doesn't have much to do with this PR.